### PR TITLE
I did task 1-5

### PR DESCRIPTION
**Pull request by aklutskaya**

### CODE REVIEW

File: src/homework/day8/ButterfliesTask.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        }
+        System.out.println();
+
+        int count = 0;
+        for (String element : butterflies) {
+            if (element.contains("o")) {
+                count++;
+            }
+        }
+        System.out.print(count);
```
Comment (IrynaAlesiyuk) [inline src/homework/day8/ButterfliesTask.java]: It is better to add some explanation what is "count" while printing to console

File: src/homework/day8/ButterfliesTask.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        System.out.println();
+
+        for (int i = 0; i < butterflies.size(); i++) {
+            System.out.print(butterflies.get(i) + " ");
+        }
+        System.out.println();
+
+        for (String fly : butterflies) {
+            System.out.println(fly + " ");
```
Comment (IrynaAlesiyuk) [inline src/homework/day8/ButterfliesTask.java]: Why do you add "  " when printing?

File: figure.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Овал-Прямоугольник-Круг-Квадрат-Эллипс-
```
Comment (IrynaAlesiyuk) [inline figure.txt]: Try to remove '-' after the last word

File: src/homework/day8/FiguresTask.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        }
+        System.out.print(count);
+        System.out.println();
+
+        for (int i = 0; i < arrayFigures.size(); i++) {
+            System.out.print(figures.get(i) + " ");
+        }
+        System.out.println();
+
+        arrayFigures.add(3, "Треугольник");
```
Comment (IrynaAlesiyuk) [inline src/homework/day8/FiguresTask.java]: haven't had error UnsupportedOperationException here while running

File: src/homework/day8/FiguresTask.java [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+        for (String f : arrayFigures) {
```
Comment (IrynaAlesiyuk) [inline src/homework/day8/FiguresTask.java]: pls use understandable variable name instead of 'f'

File: src/homework/day8/CarTask.java [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+        try (FileWriter writer = new FileWriter("cars.txt")) {
+            for (String car : cars) {
+                writer.write("\"" + car + "\"\n");
+            }
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+        }
+
+        Iterator<String> iterator = cars.iterator();
+        while (iterator.hasNext()) {
```
Comment (IrynaAlesiyuk) [inline src/homework/day8/CarTask.java]: I think here you should remove data from list, not from file

File: src/homework/day8/ButterfliesTask.java (hunk 1 above)
Comment (aklutskaya) [reply src/homework/day8/ButterfliesTask.java]: Add explanation

File: src/homework/day8/ButterfliesTask.java (hunk 2 above)
Comment (aklutskaya) [reply src/homework/day8/ButterfliesTask.java]: It was my mistake. I deleted it

File: figure.txt (hunk 3 above)
Comment (aklutskaya) [reply figure.txt]: fixed it

File: src/homework/day8/FiguresTask.java (hunk 4 above)
Comment (aklutskaya) [reply src/homework/day8/FiguresTask.java]: changed it through set()

File: src/homework/day8/CarTask.java (hunk 6 above)
Comment (aklutskaya) [reply src/homework/day8/CarTask.java]: Changed this trough for each

File: src/homework/day8/FiguresTask.java (hunk 5 above)
Comment (aklutskaya) [reply src/homework/day8/FiguresTask.java]: changed the name

**Pull request merged**